airgun/views/partitiontable.py

```diff
@@ -0,0 +1,72 @@
+from widgetastic.widget import (
+    Checkbox,
+    Text,
+    TextInput,
+    View,
+)
+
+from airgun.views.common import (
+    BaseLoggedInView,
+    SatTab,
+    SearchableViewMixin,
+)
+from airgun.widgets import (
+    ACEEditor,
+    ActionsDropdown,
+    FilteredDropdown,
+    MultiSelect
+)
+
+
+class PartitionTableView(BaseLoggedInView, SearchableViewMixin):
+
+    title = Text("//h1[text()='Partition Tables']")
+    new = Text("//a[contains(@href, '/ptables/new')]")
+    actions = ActionsDropdown("//td//div[contains(@class, 'btn-group')]")
+    edit = Text("//a[contains(@href, 'edit') and contains(@href, 'ptables')]")
+
+    @property
+    def is_displayed(self):
+        return self.browser.wait_for_element(
+            self.title, exception=False) is not None
+
+
+class PartitionTableEditView(BaseLoggedInView):
+    name = TextInput(locator="//input[@id='ptable_name']")
+    default = Checkbox(id='ptable_default')
+    snippet = Checkbox(id='ptable_snippet')
+    os_family = FilteredDropdown(id='s2id_ptable_os_family')
+    template = ACEEditor()
+    audit_comment = TextInput(id="ptable_audit_comment")
+    submit = Text('//input[@name="commit"]')
+
+    @View.nested
+    class locations(SatTab):
+        TAB_NAME = 'Locations'
```

Comment:
It's redundant - if no TAB_NAME specified, it's set to capitalized class name ('Locations' in this case). Explicitly setting TAB_NAME is only required for tabs with name different from class name (e.g. when tab name contains space).
Same for organizations and help tabs.

Reply:
in os and activation key are used TAB_NAME 's, but names are the same with the class name should I fix it ?